downstream/assemblies/central-auth/assembly-central-auth-add-user-storage.adoc

```diff
@@ -9,7 +9,7 @@
 
 .Procedure
 . Log in to {AAPCentralAuth} as an SSO admin user.
-. Under the Configure section on the side navigation bar, click *User Federation*.
+. Under the Configure section on the side navigation bar, click btn:[User Federation].
```

Comment:
I think this is actually a menu item and violates accessibility by using a directional statement too with "side navigation". I suggest rewording to something like the following:

From the navigation bar, select menu:Configure section[User Federation].